### DISCUSSION TITLE
Song select F2 button work different with stable when pressing

### DISCUSSION THREAD

**Discussion by zvyap** [ppy/osu#25353] [Q&A] [answered]
In stable, if you hold the F2 random button, it will go to the target map with some kind of slow mode
But in lazer, it didn't happen

This feature allows me to peek the maps around(up/down) the target map, so I can just press left/right button to select it after releasing the F2 button. It slightly adds some speed to my song selection (usually I go into another map in few seconds after the last play).

Also it add some "what song is coming next" feeling when holding the F2 button

Is there any plan to add this back?

**Comment by peppy** [comment, marked as answer]
I'd like to restore the old behaviour, as it feels good. It hasn't been done yet to keep things simple.

**Reply by zvyap** [reply]
Alright, thanks for the reply, have a good day 👍
